docs/advanced/developer-guide.md

```diff
@@ -125,6 +127,8 @@ makefile overrides.
 | BASE_IMAGE_FULL            | Container base image for target image full (--target full)        | debian:buster-slim
 | BASE_IMAGE_MINIMAL         | Container base image for target image minimal (--target minimal)  | gcr.io/distroless/base
 
+
+
```

Comment:
mdlint hates extra empty lines 😄 

```suggestion
```

Reply:
Yeah I should take a look at my warnings .... Sorry about that!